lesson4/backend28.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+const express= require('express')
+const app = express();
+app.listen(3618);
+app.use(express.static('public'));
+
+app.get('/send',(req,res)=>{
+  const firstdata = req.query.thenames;
+  const seconddata = req.query.thecomments;
+  fs.writeFile('/home/kinue/garageScript/lesson4/public/storing.txt', firstdata + seconddata)
```

Comment:
put space and colon between firstdata +': '+ seconddata +'\n'. 
OR 
Optional: const nameComment = `${req.query.name}: ${req.query.comment}\n`

Reply:
done! thanks